go/vt/vtgate/engine/plan.go

```diff
@@ -37,7 +38,7 @@ type Plan struct {
 	Instructions Primitive               // Instructions contains the instructions needed to fulfil the query.
 	BindVarNeeds *sqlparser.BindVarNeeds // Stores BindVars needed to be provided as part of expression rewriting
 	Warnings     []*query.QueryWarning   // Warnings that need to be yielded every time this query runs
-	TablesUsed   []string                // TablesUsed is the list of tables that this plan will query
+	TablesUsed   sqlparser.TableNames    // TablesUsed is the list of tables that this plan will query
```

Comment:
This change was made to enable `planbuilder` to figure out which keyspaces the original plan touches, and use that information to figure out whether/how to build a mirror plan, without having to do `strings.Split`.